packages/x-date-pickers/src/internals/hooks/useField/useField.utils.ts

```diff
@@ -611,7 +611,7 @@ export const getSectionsBoundaries = <TDate extends PickerValidDate>(
     }),
     meridiem: () => ({
       minimum: 0,
-      maximum: 0,
+      maximum: 1,
```

Comment:
If it will impact something I don't know - we can look for a different implementation, but it did not seem nice to have incorrect ranges (`aria-valuemin` and `aria-valuemax`) for `meridiem` token.

Reply:
The boundaries are only applied for digit editing, which does not exist for meridiem so we should be fine here